integrations/client/test_delphi_epidata.py

```diff
@@ -275,7 +275,7 @@ def test_covidcast_meta(self):
     # 2nd issue: 1 11 21
     # 3rd issue: 2 12 22
     rows = [
-      self._make_placeholder_row(time_value=self.DEFAULT_TIME_VALUE + t, issue=self.DEFAULT_ISSUE + i, value=t*10 + i)[0]
+      CovidcastRow(time_value=2020_02_02 + t, issue=2020_02_02 + i, value=t*10 + i)
```

Comment:
Not a fan of replacing with constant `2020_02_02` here, since it has to sync up with lines 302, 303, 309 below

Reply:
My main preference was having the variable easily viewable nearby and not have to go to CovidcastBase for it. How about making a local DEFAULT_TIME_VALUE, DEFAULT_ISSUE variables in this function instead?